src/praisonai/praisonai/gateway/client.py

```
@@ -0,0 +1,409 @@
+                await self._connect_once()
+                
+                # Reset reconnect attempts on successful connection
+                self._reconnect_attempts = 0
+                
+                # Start receive loop
+                self._receive_task = asyncio.create_task(self._receive_loop())
+                
+                # Wait for disconnect or stop
+                await self._receive_task
+                
+            except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)